test/reputation-mining/happy-paths.js

```diff
@@ -725,8 +725,22 @@ contract("Reputation Mining - happy paths", accounts => {
       const key = makeReputationKey(metaColony.address, new BN("2"), MINER1);
       const value = goodClient.reputations[key];
       const [branchMask, siblings] = await goodClient.getProof(key);
-      const isValid = await metaColony.verifyReputationProof(key, value, branchMask, siblings, { from: MINER1 });
+      // Checking all good parameters confirms a good proof
+      let isValid = await metaColony.verifyReputationProof(key, value, branchMask, siblings, { from: MINER1 });
       assert.isTrue(isValid);
+
+      const badKey = makeReputationKey("0xdeadbeef", new BN("2"), MINER1);
+      // Check using a bad key confirms an invalid proof
```

Comment:
You're checking bad key and a bad branchmask in this test, but not bad value or bad siblings?